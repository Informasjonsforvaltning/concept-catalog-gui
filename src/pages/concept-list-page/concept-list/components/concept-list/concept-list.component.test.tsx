import React from 'react';
import { shallow } from 'enzyme';
import { ConceptList } from './index';
import conceptItems from '../../../../../mock/concepts.json';

test('should render ConceptList', (): void => {
  const wrapper = shallow(
    <ConceptList items={conceptItems} catalogId='986105174' />
  );
  expect(wrapper).toMatchSnapshot();
});
