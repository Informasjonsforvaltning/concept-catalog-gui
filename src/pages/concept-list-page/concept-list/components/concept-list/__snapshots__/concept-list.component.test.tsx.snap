// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ConceptList 1`] = `
<div
  className="d-flex flex-column flex-fill"
>
  <Memo(ConceptListHeader)
    onSortField={[Function]}
    sortDirection="asc"
    sortField=""
  />
  <ListItem
    col1="begrep med omfang"
    col2="Dokument"
    col3="Gyldig"
    key="083c8716-6006-428f-be3b-39bc01dcc58b-0"
    path="986105174/083c8716-6006-428f-be3b-39bc01dcc58b"
    status="utkast"
  />
</div>
`;
